Libraries/Vibration/Vibration.js

```diff
@@ -98,9 +98,9 @@ const Vibration = {
   /**
    * Stop vibration
    *
-   * See https://reactnative.dev/docs/vibration.html#cancel
+   * See https://reactnative.dev/docs/vibration#cancel
    */
-  cancel: function () {
+  cancel: function() {
```

Comment:
same as above

Reply:
@cortinico thanks for the review! fixed the format